src/Cards.js

```diff
@@ -0,0 +1,39 @@
+import React from "react";
+import infoForCards from "./data/infoForCards.json";
+import "./Cards.css";
+
+const Cards = () => {
+  return (
+    <section>
+      <div className="card-deck tourist-cards">
+        {infoForCards.map((info, index) => (
```

Comment:
This is really excellent! 👏🏻 Really happy to see that you realised that the cards were repeated like this 👍🏻